classes/identity/Identity.php

```diff
@@ -376,6 +377,31 @@ public function setBiography($biography, $locale)
     {
         $this->setData('biography', $biography, $locale);
     }
+
+    /***
+     * Get the initials that should be displayed when representing the user.
+     */
+    public function getDisplayInitials(): string
+    {
+        $initials = $this->getPreferredAvatarInitials(null);
+
+        if (!$initials) {
+            // Get the individual name parts. Using hyphens and spaces as delimiters
+            $words = collect(preg_split('/[\s-]+/', $this->getFullName()));
```

Comment:
Has there been any discussion around best practice for selecting the initials when they aren't provided by the `getPreferredAvatarInitials()` above? The code here looks fine, but I'm wondering if there are any other established practices or discussions had elsewhere (I suppose that's what the preferred avatar initials are for 😅).

Reply:
On further revision, I've found an earlier [issue](https://github.com/pkp/pkp-lib/issues/8899#issue-1657535800) outlining how initials should work based on the user's name; this is listed under the **_Specs Update - Tuesday, October 1st, 2024_** section. 

I'll review the code and make the necessary chnages.